4337/contracts/Safe4337Module.sol

```diff
@@ -20,28 +20,56 @@ import {ISafe} from "./interfaces/Safe.sol";
  *      - Replay protection is handled by the entry point.
  */
 contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandler {
+    /**
+     * @dev A structure used internally for computing the EIP-712 signing message for a Safe operation.
+     */
+    struct SafeOpFields {
```

Comment:
Minor nitpick: I feel that a name like `EncodedSafeOpType` or similar might be more descriptive ...

Reply:
I ended up calling it `EncodedSafeOpStruct` as this is used in [the `hashStruct`](https://eips.ethereum.org/EIPS/eip-712#definition-of-hashstruct) algorithm (while [`encodeType`](https://eips.ethereum.org/EIPS/eip-712#definition-of-encodetype) already has a special meaning - encoding the type definition used in computing the type hash).

Reply:
ca530e28a16a649a35906a8bd7ed20889e351065